packages/gatsby-plugin-vercel-builder/README.md

```diff
@@ -0,0 +1,20 @@
+# @vercel/gatsby-plugin-vercel-builder
+
+This plugin generates [Vercel Build Output API v3](https://vercel.com/docs/build-output-api/v3) for Gatsby v4 projects.
+
+The Vercel platform automatically injects this plugin for you if it can detect Gatsby v4 in your project's dependency list. If this is the case, you should see a log message in your project's [build logs](https://vercel.com/docs/concepts/deployments/logs#build-logs) such as:
```

Comment:
```suggestion
The Vercel platform automatically injects this plugin for you if it can detect Gatsby v4+ in your project's `package.json` dependencies. If detected, you will see a log message in your project's [build logs](https://vercel.com/docs/concepts/deployments/logs#build-logs) as follows:
```